.github/CODEOWNERS

```diff
@@ -18,31 +18,31 @@ pkg/postgres/**/*                   @stackrox/core-workflows
 tests/upgrade/*                     @stackrox/core-workflows
 pkg/search/**/*                     @stackrox/core-workflows
 
-# Merlin's main responsibilities include roxctl, authN (authproviders), authZ (SAC), declarative config,
-# performance testing, administration events, cloud sources, and image signatures.
-*/administration/**/*       @stackrox/merlin
-*/auth/**/*                 @stackrox/merlin
-*/cloudsources/**/*         @stackrox/merlin
-*/declarativeconfig/**/*    @stackrox/merlin
-*/discoveredclusters/**/    @stackrox/merlin
-*/signatureintegration/**/* @stackrox/merlin
-pkg/features/**/*           @stackrox/merlin
-pkg/sac/**/*                @stackrox/merlin
-pkg/signatures/**/*         @stackrox/merlin
-roxctl/**/*                 @stackrox/merlin
-tests/performance           @stackrox/merlin
```

Comment:
Are you sure you didn't use any special characters? This how it renders in "my" GitHub.
<img width="1083" alt="Screenshot 2025-02-19 at 11 29 25" src="https://github.com/user-attachments/assets/2d872f4d-e82b-4576-bf9b-c7cfe9d73521" />

Reply:
Looks the same for me. I believe it's a Github glitch on commit it looks OK https://github.com/stackrox/stackrox/commit/689c3e331f418fe9adfcd05ae5d5d14bcd8f116b
 For some reason it does not recognize @stackrox/sensor-ecosystem  team :(

Reply:
> Unknown owner on line 23: make sure the team @stackrox/sensor-ecosystem exists, is publicly visible, and has write access to the repository

Reply:
@rukletsov Could you add repo permissions to the team? Or we can merge it as is and see if this will work

Reply:
Hmmm, let me check this again.

Reply:
Done!